release/release_test.go

```diff
@@ -165,3 +165,26 @@ func (s *ReleaseTestSuite) TestWSL(c *C) {
 
 	c.Check(release.IsWSL(), Equals, true)
 }
+
+func (s *ReleaseTestSuite) TestSystemctlSupportsUserUnits(c *C) {
+	reset := release.MockReleaseInfo(&release.OS{
+		ID: "distro-id",
+	})
+	defer reset()
+	c.Check(release.SystemctlSupportsUserUnits(), Equals, true)
+
+	// Ubuntu 14.04's systemd is too old for user units
+	reset = release.MockReleaseInfo(&release.OS{
+		ID:        "ubuntu",
+		VersionID: "14.04",
+	})
+	defer reset()
+	c.Check(release.SystemctlSupportsUserUnits(), Equals, false)
+
+	// Other Ubuntu releasess are fine
+	reset = release.MockReleaseInfo(&release.OS{
+		ID: "ubuntu",
+	})
```

Comment:
It's testing the code path of "ubuntu with a version number anything but 14.04".  In that context, an empty string and "16.04" are equivalent.  In some ways, listing an explicit version number would make the test look more specific than it is.

Reply:
I have a mild preference for adding this case, since while right now the implementation is just `!= "14.04"` this code could always be refactored to do something more intelligent, in which case having additional unit tests already written is useful.

Reply:
I've refactored the test a bit to test a table of (distro, version) pairs now, checking a few different Ubuntu version strings that should be treated as supported.

Reply:
Thank you!